[All Hosts] (runtimes) Note about JS versions for the JS-only runtime (#4548)
OfficeDev/office-js-docs-pr: docs/testing/runtimes.md

Details about these types are later in this article at [JavaScript-only runtime](#javascript-only-runtime) and [Browser runtime](#browser-runtime).

> The following table shows which possible features of an add-in that uses each type of runtime.

Reviewer: I find this phrasing a bit confusing. Does this work? 
```suggestion
The following table shows the possible features of an add-in, according to each type of runtime.
```
